src/patternfly/layouts/Grid/grid-item.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="pf-l-grid__item{{#if grid-item--modifier}} {{grid-item--modifier}}{{/if}}"
+<div class="pf-l-grid__item{{#if grid-item--modifier}}{{grid-item--modifier}}{{/if}}"
```

Comment:
looks like you removed the space on accident.

Reply:
thanks, fixed that